src/action/common/place_nix_configuration.rs

```diff
@@ -33,31 +33,34 @@ impl PlaceNixConfiguration {
         let settings = nix_config.settings_mut();
 
         settings.insert("build-users-group".to_string(), nix_build_group_name);
-        let experimental_features = ["nix-command", "flakes", "auto-allocate-uids"];
-        match settings.entry("experimental-features".to_string()) {
-            Entry::Occupied(mut slot) => {
-                let slot_mut = slot.get_mut();
-                for experimental_feature in experimental_features {
-                    if !slot_mut.contains(experimental_feature) {
-                        *slot_mut += " ";
-                        *slot_mut += experimental_feature;
+
+        if !cfg!(feature = "nix-community") {
```

Comment:
One note is this `if !cfg(...)` doesn't do code elimination like `#[cfg(...)]` does.

Using a strategy like this would ensure the code is not built when the feature is not used.

```rust
#[cfg!(feature = "nix-community")]
{
```

Reply:
Fixed!